### PR TITLE
fixed scoreboard update for slow racers (less hacky)

### PR DESCRIPTION
**Pull request by ChillerDragon**
Updating slow times still doesn't work. But setting bad times works well without hacks.

### CODE REVIEW

File: autoexec_server.cfg [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#
+﻿#
```
Comment (heinrich5991) [inline autoexec_server.cfg]: Mh?

File: src/game/server/score/sql_score.cpp [hunk 2]
```diff
@@ -235,8 +235,11 @@ bool CSqlScore::LoadScoreThread(CSqlServer* pSqlServer, const CSqlData *pGameDat
 			float Time = (float)pSqlServer->GetResults()->getDouble("Time");
 			pData->PlayerData(pData->m_ClientID)->m_BestTime = Time;
 			pData->PlayerData(pData->m_ClientID)->m_CurrentTime = Time;
-			if(pData->GameServer()->m_apPlayers[pData->m_ClientID])
+			if (pData->GameServer()->m_apPlayers[pData->m_ClientID])
```
Comment (heinrich5991) [inline src/game/server/score/sql_score.cpp]: Nit: `if` shouldn't have a space before the paranthesis in Teeworlds code.

File: autoexec_server.cfg (hunk 1 above)
Comment (ChillerDragon) [inline autoexec_server.cfg]: idk maybe because i added and removed a rcon

File: src/game/server/score/sql_score.cpp (hunk 2 above)
Comment (ChillerDragon) [inline src/game/server/score/sql_score.cpp]: Oke. If some1 has a bad time and finishes agian with a better but still very high time (for Example 1025 -> 555) it won't update scoreboard. Idk why.

**Comment by Learath2** [comment]
Maybe instead of bolting on more hacks onto the score we could implement `NETMSG_RACETIME`.

**Comment by heinrich5991** [comment]
This is not bolting more hacks onto the score, this is removing one.

**Comment by heinrich5991** [comment]
@ChillerDragon It would be awesome if updating times worked – do you think you could debug that?

**Comment by ChillerDragon** [comment]
@heinrich5991 "It would be awesome if updating times worked" Finishing two times in a row (1025 -> 500) worked fine for me. But after reconnecting it doesn't work anymore. I guess that is caused by another bug. Loading times on joining the server seems to be broken for me. Maybe because i use win or because i use filescore and not SQL. I get my time on namechange but then the update is still not working. 

"do you think you could debug that?"
Nope, sry im still a beginner.
But it would be cool if some1 could test it in a ddnet like enviroment with sql and unix and stuff. I think it could work fine there.

**Comment by heinrich5991** [comment]
OK. Can you fix the style issues (don't rewrite `autoexec.cfg`, the `if` thing)? I'll merge it then.

If you need some help with telling git to do that, you can come around to IRC.

**Comment by heinrich5991** [comment]
Sorry, I forgot about this. :( You can ping me if that should happen again.

**Comment by ChillerDragon** [comment]
@heinrich5991 no problem c:

**Pull request merged**